documentation/api/io.strimzi.api.kafka.model.KafkaUserQuotas.adoc

```diff
@@ -0,0 +1,22 @@
+Kafka cluster allows user to enforce certain requests to control usage of the resources by clients.
```

Comment:
Should we mention that we support only user based quotas and not client ID based quotas?